frontend/src/components/home/Apply.js

```diff
@@ -1,7 +1,67 @@
 import React from 'react'
+import { Typography, Box, Grid, Button } from '@mui/material'
+
 
 function Apply() {
-    return <div>-apply component placeholder for home page-</div>
+    return <div>
+        <Box sx={{ padding: "50px", }}>
+            <Typography variant='h2' sx={{ fontFamily: "Lato", fontWeight: "medium", textAlign: "center" }}>
+                Get Involved
+            </Typography>
+            <br></br>
```

Comment:
Add more margin between heading and icons images, preferably using MUI margin top.